eLearning-courseware/eLearning.jsonld

```diff
@@ -0,0 +1,1271 @@
+{
+   "@context": "https://w3id.org/xapi/profiles/context",
+    "id": "https://w3id.org/xapi/eLearning",
+    "type": "Profile",
+    "conformsTo": "https://w3id.org/xapi/profiles#1.0",
+    "prefLabel": {
+        "en": "xAPI eLearning Courseware Profile"
+    },
+    "definition": {
+        "en": "The xAPI eLearning Courseware Profile describes a set of interactions used for level three interactive distance learning."
+    },
+    "seeAlso": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile",
+    "versions": [
+        {
+            "id": "https://w3id.org/xapi/eLearning/v1.0",
+            "generatedAtTime": "2017-10-09T20:04:39.214Z"
+        }
+    ],
+    "author": {
+        "type": "Organization",
+        "name": "Riptide Elements"
+    },
+    "concepts": [
+      {
+        "@id": "https://github.com/TryxAPI/xAPI-eLearning-Courseware-Profile/concepts/course",
+        "inScheme": "https://w3id.org/xapi/eLearning",
+        "@type": "ActivityType",
+        "prefLabel": {
+          "en": "Course"
+        },
+        "definition": {
+          "en": "A course is a series of lessons which lead a learner to a better understanding of a subject. Courses are the highest unit of teaching and contain topics."
+        },
+        "broadMatch": ["https://w3id.org/xapi/cmi5/activities/course"]
```

Comment:
I'm really glad to see use of the relationships, my hope is that those drive some powerful use cases as their adoption goes.

Reply:
Yes, this should be a very useful way to link profile concepts and improve interoperability.